config/jenkins_infra.ci.jenkins.io.yaml

```diff
@@ -669,6 +669,19 @@ controller:
               whiteB: "#FFFFFF"
               yellow: "#CDCD00"
               yellowB: "#FFFF00"
+      custom-header: |
+        unclassified:
+          customHeaderConfiguration:
+            enabled: true
+            header: "logo"
+            headerColor:
+              backgroundColor: "linear-gradient(90deg, rgba(239,192,92,1) 0%, rgba(171,28,28,1) 100%);"
+              color: "white"
+              hoverColor: "grey"
```

Comment:
I would suggest not using grey but a color that you see on the gradient in the middle